public/app/plugins/datasource/zipkin/plugin.json

```diff
@@ -3,7 +3,9 @@
   "name": "Zipkin",
   "id": "zipkin",
   "category": "tracing",
+  "executable": "gpx_zipkin",
```

Comment:
You can probably omit this for now unless there's a plan to make this external

Reply:
👍 Fixed in https://github.com/grafana/grafana/pull/96031/commits/4ac6937e050ac903047e7c7ec45ade9eb6aa8bf0